Meditation-App/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Meditation-App</title>
+</head>
+<body>
+    <div class="wrapper">
+        <video class="video" loop src=""></video>
+                <audio class="audio" loop  src="""></audio>
+        
+        <div class="buttons-minutes">
+            <button class="buttons-minutes__button">1 Minutes</button>
+            <button class="buttons-minutes__button">5 Minutes</button>
+            <button class="buttons-minutes__button">10 Minutes</button>
+        </div>
+        <div class="timer">
+            <div class="mask-left"></div>
+            <div class="timer__line">
+                <div class="mask-line"></div>
+            </div>
+            <div class="timer__display"><span class="time-count">0:00</span>
```

Comment:
1) need to format code
2) and change the class name according to BEM

Reply:
Ok